src/cli/configure/getEntryPoint.ts

```diff
@@ -42,6 +42,10 @@ export const getEntryPoint = ({
       // Support exported function targeting, e.g. `src/module.ts#callMeMaybe`
       const [modulePath] = value.split('#', 2);
 
+      if (!modulePath) {
+        return `${chalk.bold(value)} is an invalid module path`;
+      }
```

Comment:
Is this even possible? The only way I've been able to arrive at an empty array is by doing `''.split('')`.